src/benchmarker.js

```diff
@@ -156,8 +171,8 @@ function dump(
   let code = serialized.code;
   let total = Date.now() - start;
 
-  if (code.length >= 1000 || outputFilename) {
-    let filename = outputFilename || name + "-processed.js";
+  if (code.length >= 1000 || !isUndefinedOrNull(outputFilename)) {
```

Comment:
Since outputFilename is an optional parameter we already know that it cannot be null, so outputFilename !== undefined would be sufficient.

Reply:
I have now realized that in Flow `x === undefined` is as safe as `typeof x === "undefined"`. Should I convert my changes to the first option, which I believe is more readable?